bindings/pydeck/pydeck/bindings/deck.py

```diff
@@ -42,11 +43,15 @@ def __init__(
             Initial camera angle relative to the map, defaults to a fully zoomed out 0, 0-centered map
             To compute a viewport from data, see :func:`pydeck.data_utils.viewport_helpers.compute_view`
         mapbox_key : str, default None
-            Read on initialization from the MAPBOX_API_KEY environment variable. Defaults to None if not set.
+            Read on initialization from the ``MAPBOX_API_KEY`` environment variable. Defaults to None if not set.
```

Comment:
It's a little weird that this isn't the same environment variable as JS uses. So I need both `MapboxAccessToken` and `MAPBOX_API_KEY` to be set in my environment, but maybe not ideal to change now.